yaptide/simulation_runner/shieldhit_runner.py

```diff
@@ -10,95 +10,31 @@
 import pathlib
 
 from pymchelper.executor.options import SimulationSettings
-from pymchelper.executor.runner import OutputDataType, Runner
+from pymchelper.executor.runner import Runner as SHRunner
 
-input_cfg_templ = {}
-input_cfg_templ['beam.dat'] = """
-RNDSEED      	89736501     ! Random seed
-JPART0       	2            ! Incident particle type
-TMAX0      	{energy:3.6f}   0.0  ! Incident energy; (MeV/nucl)
-NSTAT       {nstat:d}    -1 ! NSTAT, Step of saving
-STRAGG          2            ! Straggling: 0-Off 1-Gauss, 2-Vavilov
-MSCAT           2            ! Mult. scatt 0-Off 1-Gauss, 2-Moliere
-NUCRE           0            ! Nucl.Reac. switcher: 1-ON, 0-OFF
-"""
-input_cfg_templ['mat.dat'] = """
-MEDIUM 1
-ICRU 276
-END
-"""
-input_cfg_templ['detect.dat'] = """
-Geometry Cyl
-    Name ScoringCylinder
-    R 0.0 10.0 {cyl_nr:d}
-    Z 0.0 30.0 {cyl_nz:d}
+from ..converter.converter.converter import DummmyParser
+from ..converter.converter.converter import Runner as ConvertRunner
 
-Geometry Mesh
-    Name MyMesh_YZ
-    X -5.0  5.0    {mesh_nx:d}
-    Y -5.0  5.0    {mesh_ny:d}
-    Z  0.0  30.0   {mesh_nz:d}
 
-Output
-    Filename mesh.bdo
-    Geo ScoringCylinder
-    Quantity Dose
-"""
-input_cfg_templ['geo.dat'] = """
-*---><---><--------><------------------------------------------------>
-    0    0           protons, H2O 30 cm cylinder, r=10, 1 zone
-*---><---><--------><--------><--------><--------><--------><-------->
-  RCC    1       0.0       0.0       0.0       0.0       0.0      30.0
-                10.0
-  RCC    2       0.0       0.0      -5.0       0.0       0.0      35.0
-                15.0
-  RCC    3       0.0       0.0     -10.0       0.0       0.0      40.0
-                20.0
-  END
-  001          +1
-  002          +2     -1
-  003          +3     -2
-  END
-* material codes: 1 - liquid water (ICRU material no 276), 1000 - vacuum, 0 - black body
-    1    2    3
-    1 1000    0
-"""
-
-
-def run_shieldhit(param_dict):
+def run_shieldhit(param_dict, json_to_convert):
```

Comment:
Change `json_to_convert` to something else (eg. input_dict). `json_to_conver` suggest that it is the runners responsibility to convert the input. It is also confusing that the runner has no explicit input data.

Also consider changing `param_dict` to `**kwargs`.

Reply:
Used `json_to_conver`

Reply:
I don't see how introducing a typo would make this more readable.